.github/workflows/build-image.yml

```diff
@@ -0,0 +1,39 @@
+# Triggered by Pull Request or Manually (from GitHub UI) events
+
+name: Publish image to Registry
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Push to Registry
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2.3.1
+
+    - name: Prepare tag
+      id: prepare
+      run: |
+        commit_id=$(git rev-parse --short HEAD)
```

Comment:
as discussed during the weekly meeting - bash doesn't look good for performing actions.
on the other side we have a goal to get something running in a couple of weeks so let's proceed with this simple script - possibly move it to actions to get a uniform running

Reply:
moved to actions